megamek/src/megamek/client/ui/swing/MechViewPanel.java

```diff
@@ -137,7 +137,7 @@ public void setMech(Entity entity, TROView troView){
         txtMek.setCaretPosition(0);
         Image image = FluffImageHelper.getFluffImage(entity);
         icon = null;
-        if(null != image) {
+        if(image != null) {
```

Comment:
Why are you changing the order? The habit of putting the constant first is deliberate.

Reply:
Readability of code. Having the constant first protects against bad coding habits and typos, but significantly increases the difficulty of reading the code.

This way is the preferred styling for Java, as shown in most standard coding guides.

Reply:
We generally avoid stylistic changes outside the immediate area being changed, if only to decrease the cognitive burden reviewing changes.

That said, as a long time C/C++ programmer, I probably introduced a lot of those.

Reply:
It only affects the readability of the code if you haven't gotten used to it yet. Having the constant second is more typical, but it is not best practice.

Reply:
Modern best practice is to make it readable, and code review to catch any issues with assignment versus a comparison.
Also, constant first is normally only used for ==, not != comparisons, as issues are more likely to occur there.